app.py

```diff
@@ -30,24 +34,89 @@ def __init__(self):
         super().__init__()
 
     def _appmetadata(self):
-        # see https://sdk.clams.ai/autodoc/clams.app.html#clams.app.ClamsApp._load_appmetadata
-        # Also check out ``metadata.py`` in this directory.
-        # When using the ``metadata.py`` leave this do-nothing "pass" method here.
-        pass
+        metadata = AppMetadata(
+            name="DatimexExtraction",
+            description="Extracts date and time expressions from raw text input.",
+            app_version="1.0.0",
+            app_license="Apache 2.0"
+        )
+        metadata.add_input(DocumentTypes.TextDocument)
+        metadata.add_output(AnnotationTypes.TimeExpression)
+        metadata.add_parameter(
+            name="regex",
```

Comment:
let's move the metadata block to `metadata.py` and for now, let's not expose the regex as a customizable parameter yet.